src/extensions/default/NavigationAndHistory/main.js

```diff
@@ -0,0 +1,402 @@
+/*
+ * Copyright (c) 2012 Adobe Systems Incorporated. All rights reserved.
```

Comment:
@sprintr The theme used for this dialog is inline with the theme used in the project panel (side bar). That is why this dialog has a different look than the rest of the dialogs.